apollo-portal/src/main/resources/static/styles/common-style.css

```diff
@@ -534,18 +534,19 @@ table th {
 
 /*搜索框*/
 ::-webkit-scrollbar {
```

Comment:
It seems the best way is to remove the -webkit-scrollbar related styles?

Reply:
yep. it can keep the same style of the scrollbar in different browser.

Reply:
There is a similar pr #3936 which simply removes the -webkit-scrollbar related styles and I think it also works well. We may need to decide which way to go. @lepdou do you have any suggestion?

Reply:
[#3936](https://github.com/apolloconfig/apollo/pull/3936) that's my colleague.  It's good enough for me.